tsconfig-base.json

```diff
@@ -0,0 +1,38 @@
+// Base compiler options and all compilation entry points.
```

Comment:
`tsconfig-base.json` is a common naming convention (e.g. [in tsc itself](https://github.com/microsoft/TypeScript/blob/add85e182810a3922087bb10679ebea39e0cded0/src/tsconfig-base.json)) but not attached to anything and happy to bikeshed.

Reply:
hmm, looks like rollup really doesn't like a json file not named `tsconfig.json` with comments in it

Reply:
oh, no, it's just that it can't process the `extends` entry. @adamraine I think this is `@rollup/plugin-typescript@^2.1.0` come back to haunt us :)

Reply:
could it be https://github.com/rollup/plugins/issues/243

Reply:
I think that's part of why we couldn't get the latest `@rollup/plugin-typescript` versions working, but now the old versions are broken due to predating `extends` in tsconfigs. We could drop `extends` just from `flow-report/tsconfig.json` but I think we can get the latest plugin-typescript working.

Reply:
I'll update when #13075 is landed

Reply:
> I'll update when #13075 is landed

ok, flow-report building should be good now